pom.xml

```diff
@@ -93,7 +93,32 @@
             <plugin>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.22.1</version>
-            </plugin>
+                <configuration>
+                    <excludes>
+                        <exclude>**/integration/**</exclude>
+                    </excludes>
+                </configuration>
+             </plugin>
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
from what I've read, its preferred to run integration tests with the failsafe plugin